internal/config/cgmgr/stats_linux.go

```diff
@@ -37,13 +37,19 @@ type MemoryStats struct {
 	KernelTCPUsage  uint64
 	SwapUsage       uint64
 	SwapLimit       uint64
+	// Amount of cached filesystem data mapped with mmap()
+	FileMapped uint64
+	// The number of memory usage hits limits. For cgroup v1 only
```

Comment:
```suggestion
	// The number of memory usage hits limits. For cgroup v1 only.
```

Reply:
how many times the process reached it's memory limit. in the comment above, I used the kernel [docs](https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt) definition for  `memory.failcnt`. I can change it if you like since it seems a bit vague.